src/app/hooks/useClickTrackerHandler/index.jsx

```diff
@@ -15,7 +15,11 @@ const useClickTrackerHandler = (props = {}) => {
   const href = path(['href'], props);
   const format = path(['format'], props);
 
-  const { enabled: eventTrackingIsEnabled } = useToggle('eventTracking');
+  const { enabled: eventTrackingIsEnabled, value } = useToggle('eventTracking');
+  const isExcluded =
+    typeof value === 'string'
+      ? value?.trim().split(',').includes(componentName)
+      : false;
```

Comment:
Just a personal preference here but I'd suggest to make this slightly more readable:
- add a comment to show that the `eventTracking` toggle contains a comma seprated list of components excluded from tracking
- seperate the conversion of the comma seperated list from the exclusion logic, something like:
```javascript
  const { enabled: eventTrackingIsEnabled, value } = useToggle('eventTracking');
  const excludedComponents = value?.trim().split(',');
  const isExcluded = excludedComponents.includes(componentName);
```

Reply:
Yeah I guess it is a little hard to read, can definitely make that change 👍